examples/calc/README.md

```diff
@@ -0,0 +1,11 @@
+# Lexing a simple calculator language
+
+This directory contains a very simple example of a calculator example in
+`lrlex`. Executing `cargo run` processes `src/calc.l` at compile-time;
```

Comment:
I think the second sentence here is missing a word?

Reply:
Do you mean the "executing" sentence? It reads OK (albeit it's terse) to me, but I'm biased!

Reply:
No, the one before, and now I read it again I think it has an extra word:
`This directory contains a very simple example of a calculator example in lrlex`.
Should be:
`This directory contains a very simple example of a calculator lexer in lrlex`.
or similar...?

Reply:
Agreed.